cads_adaptors/tools/url_tools.py

```diff
@@ -30,13 +31,20 @@ def requests_to_urls(
 
 def try_download(urls: List[str], context: Context, **kwargs) -> List[str]:
     paths = []
+    context.write_type = "stdout"
     for url in urls:
         path = urllib.parse.urlparse(url).path.lstrip("/")
         dir = os.path.dirname(path)
         if dir:
             os.makedirs(dir, exist_ok=True)
         try:
-            multiurl.download(url, path, **kwargs)
+            with tqdm(
```

Comment:
I know this is still a WIP, but in the final version could you add a comment here explaining that this is required to stream progress bar to the logs, so that we know for future adaptor developments.

Reply:
with the last version the change is pretty self-explanatory